Src/Backend/foodApi/Dockerfile

```diff
@@ -1,15 +1,61 @@
-# Take from https://medium.com/@uxioandrade/implementing-session-caching-in-go-with-gin-sessions-redis-and-docker-838608f6cca
-# Last build size was 996MB, which having 809MB as golang:latest
-FROM golang:latest
-
+# https://blog.golang.org/docker
+# Start from a Debian image with the latest version of Go installed
+# and a workspace (GOPATH) configured at /go.
+# FROM s390x/golang:1.14-alpine
+############################
+# STEP 1 build executable binary
+############################
+# golang alpine 1.13.5
+FROM golang@sha256:0991060a1447cf648bab7f6bb60335d1243930e38420bee8fec3db1267b84cfa as builder
```

Comment:
Remove the hashed tag. Replace with a version number or latest

Reply:
using a trusted docker image provides better security could also use golang:alpine
I have removed the comments